client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -291,7 +293,10 @@ const Checkout = React.createClass( {
 			} );
 		}
 
-		if ( cart.create_new_blog && receipt && isEmpty( receipt.failed_purchases ) ) {
+		if (
+			( cart.create_new_blog && receipt && isEmpty( receipt.failed_purchases ) ) ||
+			( isDomainOnly && cartItems.hasPlan( cart ) )
+		) {
 			notices.info(
 				this.translate( 'Almost done…' )
 			);
```

Comment:
Nice job finding this! But it does not solve the issue for me :) The reason is that on line 304, `domainName` is `null`, because there's no domain in the cart. There's only the plan - which we don't associate with the domain (at least not at this point). So it seems you need to get the domain from somewhere else.. or can you fetch the sites without the domain name?